cmake/macros/Python.cmake

```diff
@@ -48,8 +48,11 @@ function(add_python_target target)
   string(REPLACE "." "/" slash_namespace "${namespace}")
 
   add_library(${target} ${MODULE_TYPE} ${sources})
-  target_link_libraries(${target} PKG::PYTHON)
-
+  if (ANDROID)
+    target_link_libraries(${target} "python${PYMAJOR}.${PYMINOR}" "ffi" "z"  "bz2" "lzma")
+  else()
+    target_link_libraries(${target} PKG::PYTHON)
+  endif()
```

Comment:
I don't think this belongs here, is there a way to make find_package do the right thing for Python?

Reply:
Well, it's not only python linking, there you could have locale fix too for ndk and old api unless you have a better place for it.  i'm also not sure find_package in cmake ndk toolchain does even know what python is a thing :unsure:

Reply:
maybe it would be simpler to pass whole python link command via a cmake parameter. what do you think ?

Reply:
There probably already is a setting for this, such as PYTHON_LINK_LIBRARIES, or something like that

Reply:
That would be a setting containing  sysconfig.get_config_vars('LIBS')  but i've found nothing like it even in latest cmake https://cmake.org/cmake/help/v3.19/module/FindPython3.html.